tests/test_orient_symmetric.py

```diff
@@ -312,6 +338,25 @@ def testCommonLines(self, order):
         self.assertTrue(within_5 > 0.99)
         self.assertTrue(within_1 > 0.95)
 
+    def testCompleteThirdRow(self):
+        cl_orient_ests = self.cl_orient_ests[3]
+
+        # Complete third row that coincides with z-axis
+        z = np.array([0, 0, 1])
+        Rz = cl_orient_ests.complete_third_row_to_rot(z)
+
+        # Complete random third row.
+        r3 = randn(3, seed=123)
+        r3 /= norm(r3)
+        R = cl_orient_ests.complete_third_row_to_rot(r3)
+
+        # Assert that Rz is the identity matrix.
+        self.assertTrue(np.allclose(Rz, np.eye(3)))
+
+        # Assert that R is orthogonal with determinant 1.
+        self.assertTrue(np.allclose(R @ R.T, np.eye(3)))
+        self.assertTrue(np.allclose(det(R), 1))
+
```

Comment:
This is really cool. We don't need this for c4? Or it doesn't make a difference

Reply:
Oops. This is a `@staticmethod`. I was mistakenly calling it from the instance level instead of the class level. Thanks! Fixed.